tools/params_processors/link_params_processor.py

```diff
@@ -36,10 +36,22 @@ def _main(
         fp.write(f'{result}\n')
 
 
-def _parse_args(args: List[str]) -> List[str]:
-    if args[0].startswith("@"):
-        with open(args[0][1:], encoding = "utf-8") as fp:
-            return fp.read().splitlines()
+def _parse_args(args_files: List[str]) -> List[str]:
+    args = []
+    for args_path in args_files:
+        # Each argument is a path to a file containing the actual arguments
+        with open(args_path, encoding = "utf-8") as fp:
+            lines = fp.read().splitlines()
+            if lines[0].startswith("@"):
+                # Sometimes those arguments might be also be a redirect
+                with open(lines[0][1:], encoding = "utf-8") as f:
+                    args.extend(f.read().splitlines())
+            else:
+                # First argument is the tool name
+                args.extend(lines[1:])
+
+    print(args)
```

Comment:
nit : probably delete this print unless it's here for debugging purposes? @brentleyjones

Reply:
Thanks, I'll remove.